From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <quh4gko8.vanadium@gmail.com>
Date: Fri, 7 Nov 2025 07:00:17 +0000
Subject: [PATCH] Add DuckDuckGo on all regional list of search engines

Commands used for appending and replacing regional list of search
engines in json, in order:

```bash
    sed -ri "N;N;s/,(|\n)?\n        \"\&duckduckgo\"//g;P;D" definitions/regional_settings.json

    sed -ri "s/\"search_engines\": \[/&\n        \"\&duckduckgo\",/g" definitions/regional_settings.json

```
---
 definitions/regional_settings.json | 268 +++++++++++++++--------------
 1 file changed, 137 insertions(+), 131 deletions(-)

diff --git a/definitions/regional_settings.json b/definitions/regional_settings.json
index 7bdc719..e9a5406 100644
--- a/definitions/regional_settings.json
+++ b/definitions/regional_settings.json
@@ -6,39 +6,39 @@
     "AE": {
       // United Arab Emirates
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&yandex_ru",
-        "&duckduckgo"
+        "&yandex_ru"
       ]
     },
     "AL": {
       // Albania
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&yandex_tr",
-        "&duckduckgo"
+        "&yandex_tr"
       ]
     },
     "AR": {
       // Armenia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_ar",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "AT": {
       // Austria
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&ecosia",
         "&bing",
         "&yahoo_at",
@@ -49,29 +49,29 @@
     "AU": {
       // Australia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_au",
-        "&duckduckgo",
         "&ecosia"
       ]
     },
     "BA": {
       // Bosnia and Herzegovina
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "BE": {
       // Belgium
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&ecosia",
         "&bing",
         "&qwant",
@@ -82,9 +82,9 @@
     "BG": {
       // Bulgaria
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&yep",
         "&ecosia",
@@ -95,69 +95,69 @@
     "BH": {
       // Bahrain
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "BI": {
       // Burundi
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&yandex_ru",
-        "&duckduckgo"
+        "&yandex_ru"
       ]
     },
     "BN": {
       // Brunei
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "BO": {
       // Bolivia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "BR": {
       // Brazil
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_br",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "BY": {
       // Belarus
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&yandex_by",
         "&bing",
-        "&duckduckgo",
         "&mail_ru"
       ]
     },
     "BZ": {
       // Belize
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
-        "&duckduckgo",
         "&yahoo",
         "&yandex_com"
       ]
@@ -165,19 +165,19 @@
     "CA": {
       // Canada
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_ca",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "CH": {
       // Switzerland
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
-        "&duckduckgo",
         "&yahoo_ch",
         "&ecosia"
       ]
@@ -185,16 +185,17 @@
     "CL": {
       // Chile
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_cl",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "CN": {
       // China
       "search_engines": [
+        "&duckduckgo",
         "&baidu",
         "&bing",
         "&sogou",
@@ -205,29 +206,29 @@
     "CO": {
       // Colombia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_co",
-        "&duckduckgo",
         "&ecosia"
       ]
     },
     "CR": {
       // Costa Rica
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "CY": {
       // Republic of Cyprus
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&yahoo_emea",
         "&ecosia",
@@ -238,10 +239,10 @@
     "CZ": {
       // Czechia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&seznam",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&ecosia",
         "&privacywall",
@@ -251,8 +252,8 @@
     "DE": {
       // Germany
       "search_engines": [
-        "&google",
         "&duckduckgo",
+        "&google",
         "&brave",
         "&ecosia",
         "&bing",
@@ -264,9 +265,9 @@
     "DK": {
       // Denmark
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&ecosia",
         "&qwant",
@@ -277,39 +278,39 @@
     "DO": {
       // Dominican Republic
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "DZ": {
       // Algeria
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_fr",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "EC": {
       // Ecuador
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_es",
-        "&duckduckgo",
         "&ecosia"
       ]
     },
     "EE": {
       // Estonia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&ecosia",
         "&qwant",
@@ -320,20 +321,20 @@
     "EG": {
       // Egypt
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "ES": {
       // Spain
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
         "&bing",
-        "&duckduckgo",
         "&ecosia",
         "&yahoo_es",
         "&privacywall",
@@ -343,9 +344,9 @@
     "FI": {
       // Finland
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&ecosia",
         "&qwant",
@@ -356,21 +357,21 @@
     "FO": {
       // Faroe Islands
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_uk",
-        "&duckduckgo",
         "&ecosia"
       ]
     },
     "FR": {
       // France
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
         "&ecosia",
         "&bing",
-        "&duckduckgo",
         "&qwant",
         "&yahoo_fr",
         "&lilo"
@@ -379,19 +380,19 @@
     "GB": {
       // United Kingdom
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_uk",
-        "&duckduckgo",
         "&ecosia"
       ]
     },
     "GR": {
       // Greece
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&yahoo_emea",
         "&ecosia",
@@ -402,16 +403,17 @@
     "GT": {
       // Guatemala
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "HK": {
       // Hong Kong
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_hk",
@@ -422,19 +424,19 @@
     "HN": {
       // Honduras
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "HR": {
       // Croatia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&yep",
         "&ecosia",
@@ -445,10 +447,10 @@
     "HU": {
       // Hungary
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
         "&bing",
-        "&duckduckgo",
         "&ecosia",
         "&yep",
         "&yahoo_emea",
@@ -458,18 +460,18 @@
     "ID": {
       // Indonesia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&yandex_com",
         "&bing",
-        "&yahoo_id",
-        "&duckduckgo"
+        "&yahoo_id"
       ]
     },
     "IE": {
       // Ireland
       "search_engines": [
-        "&google",
         "&duckduckgo",
+        "&google",
         "&brave",
         "&bing",
         "&yahoo_uk",
@@ -481,48 +483,48 @@
     "IL": {
       // Israel
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yandex_ru",
-        "&yahoo",
-        "&duckduckgo"
+        "&yahoo"
       ]
     },
     "IN": {
       // India
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_in",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "IQ": {
       // Iraq
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yandex_tr",
-        "&yahoo",
-        "&duckduckgo"
+        "&yahoo"
       ]
     },
     "IR": {
       // Iran
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "IS": {
       // Iceland
       "search_engines": [
-        "&google",
         "&duckduckgo",
+        "&google",
         "&brave",
         "&bing",
         "&ecosia",
@@ -534,10 +536,10 @@
     "IT": {
       // Italy
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
         "&bing",
-        "&duckduckgo",
         "&ecosia",
         "&yahoo_it",
         "&qwant",
@@ -547,46 +549,47 @@
     "JM": {
       // Jamaica
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "JO": {
       // Jordan
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "JP": {
       // Japan
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&yahoo_jp",
         "&bing",
-        "&duckduckgo",
         "&ecosia"
       ]
     },
     "KE": {
       // Kenya
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "KR": {
       // South Korea
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&naver",
         "&bing",
@@ -597,16 +600,17 @@
     "KW": {
       // Kuwait
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yandex_tr",
-        "&yahoo",
-        "&duckduckgo"
+        "&yahoo"
       ]
     },
     "KZ": {
       // Kazakhstan
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&yandex_kz",
         "&bing",
@@ -617,18 +621,18 @@
     "LB": {
       // Lebanon
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "LI": {
       // Liechtenstein
       "search_engines": [
-        "&google",
         "&duckduckgo",
+        "&google",
         "&brave",
         "&bing",
         "&ecosia",
@@ -640,9 +644,9 @@
     "LT": {
       // Lithuania
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&ecosia",
         "&yahoo_emea",
@@ -653,9 +657,9 @@
     "LU": {
       // Luxembourg
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&ecosia",
         "&qwant",
@@ -666,9 +670,9 @@
     "LV": {
       // Latvia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&ecosia",
         "&privacywall",
@@ -679,70 +683,70 @@
     "LY": {
       // Libya
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yandex_tr",
-        "&yahoo",
-        "&duckduckgo"
+        "&yahoo"
       ]
     },
     "MA": {
       // Morocco
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_fr",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "MC": {
       // Monaco
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_fr",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "MD": {
       // Moldova
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&yandex_ru",
         "&bing",
-        "&duckduckgo",
         "&mail_ru"
       ]
     },
     "ME": {
       // Montenegro
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yandex_ru",
-        "&yahoo",
-        "&duckduckgo"
+        "&yahoo"
       ]
     },
     "MK": {
       // Macedonia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "MT": {
       // Malta
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
         "&bing",
-        "&duckduckgo",
         "&ecosia",
         "&yahoo_emea",
         "&privacywall",
@@ -752,38 +756,38 @@
     "MX": {
       // Mexico
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_mx",
-        "&duckduckgo",
         "&ecosia"
       ]
     },
     "MY": {
       // Malaysia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_my",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "NI": {
       // Nicaragua
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "NL": {
       // Netherlands
       "search_engines": [
-        "&google",
         "&duckduckgo",
+        "&google",
         "&brave",
         "&bing",
         "&ecosia",
@@ -795,8 +799,8 @@
     "NO": {
       // Norway
       "search_engines": [
-        "&google",
         "&duckduckgo",
+        "&google",
         "&brave",
         "&bing",
         "&ecosia",
@@ -808,69 +812,69 @@
     "NZ": {
       // New Zealand
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_nz",
-        "&duckduckgo",
         "&ecosia"
       ]
     },
     "OM": {
       // Oman
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "PA": {
       // Panama
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_es",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "PE": {
       // Peru
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_pe",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "PH": {
       // Philippines
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_ph",
-        "&ecosia",
-        "&duckduckgo"
+        "&ecosia"
       ]
     },
     "PK": {
       // Pakistan
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "PL": {
       // Poland
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&yahoo_emea",
         "&ecosia",
@@ -881,20 +885,20 @@
     "PR": {
       // Puerto Rico
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "PT": {
       // Portugal
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
         "&bing",
-        "&duckduckgo",
         "&ecosia",
         "&yep",
         "&qwant",
@@ -904,29 +908,29 @@
     "PY": {
       // Paraguay
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_es",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "QA": {
       // Quatar
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_tr"
       ]
     },
     "RO": {
       // Romania
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&yahoo_emea",
         "&bing",
         "&yep",
@@ -937,48 +941,48 @@
     "RS": {
       // Serbia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&yandex_ru",
-        "&duckduckgo"
+        "&yandex_ru"
       ]
     },
     "RU": {
       // Russia
       "search_engines": [
+        "&duckduckgo",
         "&yandex_ru",
         "&google",
         "&bing",
-        "&duckduckgo",
         "&mail_ru"
       ]
     },
     "RW": {
       // Rwanda
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "SA": {
       // Saudi Arabia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yandex_com",
-        "&yahoo",
-        "&duckduckgo"
+        "&yahoo"
       ]
     },
     "SE": {
       // Sweden
       "search_engines": [
-        "&google",
         "&duckduckgo",
+        "&google",
         "&brave",
         "&bing",
         "&ecosia",
@@ -990,19 +994,19 @@
     "SG": {
       // Singapore
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_sg",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "SI": {
       // Slovenia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&ecosia",
         "&yep",
@@ -1013,9 +1017,9 @@
     "SK": {
       // Slovakia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&brave",
-        "&duckduckgo",
         "&bing",
         "&seznam",
         "&ecosia",
@@ -1026,66 +1030,67 @@
     "SV": {
       // El Salvador
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_es",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "SY": {
       // Syria
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yandex_ru",
-        "&yahoo",
-        "&duckduckgo"
+        "&yahoo"
       ]
     },
     "TH": {
       // Thailand
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_th",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "TN": {
       // Tunisia
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_fr",
-        "&yandex_com",
-        "&duckduckgo"
+        "&yandex_com"
       ]
     },
     "TR": {
       // Turkey
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&yandex_tr",
         "&yahoo_tr",
-        "&bing",
-        "&duckduckgo"
+        "&bing"
       ]
     },
     "TT": {
       // Trinidad and Tobago
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "TW": {
       // Taiwan
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&yahoo_tw",
         "&bing",
@@ -1096,8 +1101,8 @@
     "TZ": {
       // Tanzania
       "search_engines": [
-        "&google",
         "&duckduckgo",
+        "&google",
         "&bing",
         "&yahoo",
         "&yandex_com"
@@ -1106,80 +1111,80 @@
     "UA": {
       // Ukraine
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&yandex_ru",
         "&bing",
-        "&duckduckgo",
         "&yahoo"
       ]
     },
     "US": {
       // United States
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "UY": {
       // Uruguay
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_es",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "VE": {
       // Venezuela
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo_es",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "VN": {
       // Vietnam
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&coccoc",
         "&yahoo",
-        "&bing",
-        "&duckduckgo"
+        "&bing"
       ]
     },
     "YE": {
       // Yemen
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "ZA": {
       // South Africa
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&yandex_com"
       ]
     },
     "ZW": {
       // Zimbabwe
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo",
-        "&duckduckgo",
         "&sogou"
       ]
     },
@@ -1189,6 +1194,7 @@
     "ZZ": {
       // Default
       "search_engines": [
+        "&duckduckgo",
         "&google",
         "&bing",
         "&yahoo"
